### ISSUE TITLE
Add guest functionality

### ISSUE THREAD

**Issue by EricPickup** [EricPickup/uwindsor-css-hub#69]
Most CSS events allow students to bring guests (non-CS students).
- add guest limit option to events
- dynamic guest fields when registering
- allow users to add guests

**Comment by HarshdipD** [comment]
@EricPickup  what's the status of this issue? I think you're already working on it as I saw codes about the guest in controllers so just wanna know if you're working on it

**Comment by EricPickup** [comment]
@HarshdipD Pretty sure I added the model and associations between a user and guest but never actually added this to the forms. Feel free to take it up

**Comment by amanpatel123** [comment]
Just want to clarify few things here, so the guest list here are all the people who are registered in this event 

![image](https://user-images.githubusercontent.com/46072181/79301226-05d23a80-7eb7-11ea-9983-6ee3f17065a9.png)

- currently, these guest lists only have CS Students but for some events we want students to also bring Non-CS students correct? Meaning for some special events this list will also have non-cs students email here?

**Comment by EricPickup** [comment]
That is correct @amanpatel123, it's low-priority but we have some events that we allow students to bring `n` amount of friends (usually 1-2). The user would type in the students name and email to add to the list as their guest.
